src/app/ApplicationTemplate.Shared/CoreStartup.cs

```diff
@@ -89,7 +89,7 @@ private async Task ExecuteInitialNavigation(CancellationToken ct, IServiceProvid
 				await sectionsNavigator.Navigate(ct, () => new OnboardingPageViewModel());
 			}
 //-:cnd:noEmit
-#if __ANDROID__ || __IOS__ || WINDOWS_UWP
+#if __IOS__ || WINDOWS_UWP
```

Comment:
How does Android know when to transition from the splashscreen to the main app now?

Reply:
with AndroidX.Core.SplashScreen.SplashScreen.InstallSplashScreen(this);

Reply:
Why wouldn't this be at the same place where we currently remove the extended splashscreen?

Reply:
Android takes over the splashscreen and the OS does the transition to the app itself

Reply:
Ok, but the app is not ready to be displayed until the first navigation. That means the splash screen needs to stay at least until the first navigation is completed.

Reply:
We will have to do another PR to handle that transition. For now, we've kept the splashscreen we used before set by windowBackground=splash.xml so we technically have two identical splashcreen one after another. 
Here is what it looks like : 
https://user-images.githubusercontent.com/84537432/178575248-b87737d5-64b1-478a-bccb-2281c95da4cd.mp4
When removing this item from the AppTheme style, we have a white screen while the app is starting and executing the initial navigation. It looks like this.
https://user-images.githubusercontent.com/84537432/178575506-deb3fee2-099f-4310-9338-5dbc2ae43147.mp4

Since we do not want two splashscreen that can differ in style, we will add another task to fix this in the app or in the ExtendedSplashScreen package.